project/pyproject.toml.jinja

```diff
@@ -28,6 +28,7 @@ pytest-cov = "^2.11.1"
 pytest-randomly = "^3.5.0"
 pytest-sugar = "^0.9.4"
 pytest-xdist = "^2.2.0"
+safety = "^1.10.3"
```

Comment:
There's an issue with this: https://github.com/advisories/GHSA-7q25-qrjw-6fg2.
Unless you implement [the solution offered here](https://github.com/pyupio/safety/issues/348), safety should stay an external executable, for example installed with pipx. Feel free to document this instead.